### ISSUE TITLE
Plugin not compatible with Linkerd 1.3.0

### ISSUE THREAD

**Issue by kumudt** [linkerd/linkerd-zipkin#15]
linkerd-zipkin plugin has to be updated to make it compatible with Linkerd 1.3.0
```
	at java.lang.Class.newInstance(Class.java:442)
	at com.twitter.app.LoadService$.$anonfun$apply$7(LoadService.scala:73)
	at scala.collection.TraversableLike.$anonfun$flatMap$1(TraversableLike.scala:241)
	at scala.collection.mutable.ResizableArray.foreach(ResizableArray.scala:59)
	at scala.collection.mutable.ResizableArray.foreach$(ResizableArray.scala:52)
	at scala.collection.mutable.ArrayBuffer.foreach(ArrayBuffer.scala:48)
	at scala.collection.TraversableLike.flatMap(TraversableLike.scala:241)
	at scala.collection.TraversableLike.flatMap$(TraversableLike.scala:238)
	at scala.collection.AbstractTraversable.flatMap(Traversable.scala:104)
	at com.twitter.app.LoadService$.apply(LoadService.scala:62)
	at com.twitter.finagle.util.LoadService$.apply(LoadService.scala:14)
	at io.buoyant.linkerd.Linker$.LoadedInitializers$lzycompute(Linker.scala:65)
	at io.buoyant.linkerd.Linker$.LoadedInitializers(Linker.scala:64)
	at io.buoyant.linkerd.Linker$.parse$default$2(Linker.scala:79)
	at io.buoyant.linkerd.Main$.loadLinker(Main.scala:61)
	at io.buoyant.linkerd.Main$.main(Main.scala:31)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.twitter.app.App.$anonfun$nonExitingMain$3(App.scala:233)
	at scala.Option.foreach(Option.scala:257)
	at com.twitter.app.App.nonExitingMain(App.scala:232)
	at com.twitter.app.App.nonExitingMain$(App.scala:210)
	at io.buoyant.linkerd.Main$.nonExitingMain(Main.scala:20)
	at com.twitter.app.App.main(App.scala:198)
	at com.twitter.app.App.main$(App.scala:196)
	at io.buoyant.linkerd.Main$.main(Main.scala:20)
	at io.buoyant.linkerd.Main.main(Main.scala)
Caused by: java.lang.IllegalArgumentException: Multiple differing Toggle config resources found for com.twitter.finagle.netty4, jar:file:/io.buoyant/linkerd/1.3.0/bundle-exec!/com/twitter/toggles/configs/com.twitter.finagle.netty4.json, jar:file:/io.buoyant/linkerd/1.3.0/plugins/linkerd-zipkin-0.0.1.jar!/com/twitter/toggles/configs/com.twitter.finagle.netty4.json
	at com.twitter.finagle.toggle.StandardToggleMap$.selectResource(StandardToggleMap.scala:163)
	at com.twitter.finagle.toggle.StandardToggleMap$.loadJsonConfigWithEnv(StandardToggleMap.scala:180)
	at com.twitter.finagle.toggle.StandardToggleMap$.loadJsonConfig(StandardToggleMap.scala:122)
	at com.twitter.finagle.toggle.StandardToggleMap$.apply(StandardToggleMap.scala:100)
	at com.twitter.finagle.toggle.StandardToggleMap$.apply(StandardToggleMap.scala:85)
	at com.twitter.finagle.netty4.package$.&lt;init&gt;(package.scala:17)
	at com.twitter.finagle.netty4.package$.&lt;clinit&gt;(package.scala)
	... 56 more
Exception thrown in main on startup
```

**Comment by wmorgan** [comment]
Thanks for the report! We'll take a look.

**Comment by klingerf** [comment]
Hey @kumudt, thanks for reporting! We're working on publishing the linkerd 1.3.0 artifacts now, and I'll update this repo to consume them as soon as they're ready.

**Comment by klingerf** [comment]
Hi @kumudt -- I've put together a fix for this issue in #16. We'll merge it soon.

**Comment by kumudt** [comment]
Thank you.
